bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/BagsResult.scala

```diff
@@ -0,0 +1,68 @@
+package edu.gemini.spModel.target.env
+
+import edu.gemini.shared.util.immutable.{Option => GOption}
+import edu.gemini.spModel.pio.{Pio, PioFactory, ParamSet}
+import edu.gemini.spModel.target.SPTarget
+import edu.gemini.spModel.rich.shared.immutable._
+
+sealed trait BagsResult extends Cloneable {
+  val id: String
+  val target: Option[SPTarget] = None
+  def targetAsJava: GOption[SPTarget] = target.asGeminiOpt
+  override def clone: BagsResult = this
+
+  override def equals(obj: scala.Any): Boolean = obj match {
+    case other: BagsResult => target.equals(other.target)
```

Comment:
Hmmm... good point. Not sure what the easiest way is to fix this with case classes of a sealed trait where I want to compare data members.
I can compare `id`s or something more elaborate.